maintainers/maintainer-list.nix

```diff
@@ -7183,6 +7183,12 @@
       fingerprint = "1412 816B A9FA F62F D051 1975 D3E1 B013 B463 1293";
     }];
   };
+  itcger = {
+    email = "nixpkgs@dirk-osburg.de";
+    github = "itc-ger";
```

Comment:
This doesn't match your current username

Reply:
That's right. I know I have to correct these things - will do this soon.